src/DescriptorsCommand.cc

```diff
@@ -535,6 +531,9 @@ int FindDescriptor::construct_protobuf(
         }
     }
 
+    results["list"].append(VDMS_DESC_LABEL_PROP);
```

Comment:
Am I right in seeing this as a cosmetic change?

Reply:
not really. is needed independent if the user specifies results with list now.